<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Code2dWindowClass</class>
 <widget class="QMainWindow" name="Code2dWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>872</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Code2dWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>591</width>
      <height>751</height>
     </rect>
    </property>
    <property name="title">
     <string>显示窗口</string>
    </property>
    <widget class="CMyLabel" name="lbl_Window">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>571</width>
       <height>501</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pBtn_ReadImage">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>650</y>
       <width>111</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>打开图片</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pBtn_FitImage">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>650</y>
       <width>111</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>自适应</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_Status">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>550</y>
       <width>561</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>坐标灰度信息</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_ClearWindow">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>650</y>
       <width>111</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>清空窗口</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="ckb_ShowObj">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>600</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>缩放时显示图形</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_Hide">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>650</y>
       <width>111</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>关闭</string>
     </property>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget_5">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>10</y>
      <width>561</width>
      <height>751</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_17">
     <attribute name="title">
      <string>参数</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>28</y>
        <width>521</width>
        <height>681</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_11">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_16">
         <item>
          <widget class="QPushButton" name="btn_DrawRoi_Code2d">
           <property name="text">
            <string>绘制识别区域(不绘制默认全图搜索)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_ClearRoi_Code2d">
           <property name="text">
            <string>清空ROI</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_29">
           <item>
            <widget class="QLabel" name="label_58">
             <property name="text">
              <string>二维码个数：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spb_CodeNum_Code2d">
             <property name="maximum">
              <number>9999</number>
             </property>
             <property name="value">
              <number>1</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_33">
           <item>
            <widget class="QLabel" name="label_65">
             <property name="text">
              <string>超时时间：</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_32">
             <item>
              <widget class="QSpinBox" name="spb_TimeOut_Code2d">
               <property name="maximum">
                <number>99999</number>
               </property>
               <property name="value">
                <number>100</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_66">
               <property name="text">
                <string>ms</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_18">
           <item>
            <widget class="QLabel" name="label_64">
             <property name="text">
              <string>解码模式：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cmb_Tolerance_Code2d">
             <item>
              <property name="text">
               <string>标准</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>增强</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer_8">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>268</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_30">
         <item>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>二维码类型</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_9">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_10">
              <item row="0" column="0">
               <layout class="QVBoxLayout" name="verticalLayout_22">
                <item>
                 <widget class="QCheckBox" name="ckb_DM_Code2d">
                  <property name="text">
                   <string>DM</string>
                  </property>
                  <property name="checked">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="ckb_Aztec_Code2d">
                  <property name="text">
                   <string>Aztec</string>
                  </property>
                  <property name="checked">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="ckb_GS1DM_Code2d">
                  <property name="text">
                   <string>GS1DM</string>
                  </property>
                  <property name="checked">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="ckb_MicroQR_Code2d">
                  <property name="text">
                   <string>MicroQR</string>
                  </property>
                  <property name="checked">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="0" column="1">
               <layout class="QVBoxLayout" name="verticalLayout_23">
                <item>
                 <widget class="QCheckBox" name="ckb_QR_Code2d">
                  <property name="text">
                   <string>QR</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="ckb_GS1Aztec_Code2d">
                  <property name="text">
                   <string>GS1Aztec</string>
                  </property>
                  <property name="checked">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="ckb_GS1QR_Code2d">
                  <property name="text">
                   <string>GS1QR</string>
                  </property>
                  <property name="checked">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="ckb_PDF417_Code2d">
                  <property name="text">
                   <string>PDF417</string>
                  </property>
                  <property name="checked">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_18">
     <attribute name="title">
      <string>过滤</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>521</width>
        <height>681</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_59">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_17">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_19">
           <item>
            <widget class="QLabel" name="label_59">
             <property name="text">
              <string>特定字符开头：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txt_HeadChar_Code2d"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_20">
           <item>
            <widget class="QLabel" name="label_60">
             <property name="text">
              <string>特定字符结尾：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txt_EndChar_Code2d"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_21">
           <item>
            <widget class="QLabel" name="label_61">
             <property name="text">
              <string>包含特定字符：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txt_ContainChar_Code2d"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_22">
           <item>
            <widget class="QLabel" name="label_62">
             <property name="text">
              <string>不包含特定字符：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txt_NotContainChar_Code2d"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_23">
           <item>
            <widget class="QLabel" name="label_63">
             <property name="text">
              <string>条码最小长度：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spb_CodeLength_Code2d">
             <property name="minimum">
              <number>2</number>
             </property>
             <property name="maximum">
              <number>999</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_9">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>198</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_165">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_28">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_24">
             <item>
              <widget class="QPushButton" name="btn_LoadCode2dData">
               <property name="text">
                <string>参数加载</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_SaveCode2dData">
               <property name="text">
                <string>参数保存</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_25">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_31">
             <item>
              <widget class="QPushButton" name="btn_FindCode2d">
               <property name="text">
                <string>运行</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lbl_CT_Code2d">
               <property name="text">
                <string>运行时间</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_19">
     <attribute name="title">
      <string>结果</string>
     </attribute>
     <widget class="QTableWidget" name="tablewidget_Results_Code2d">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>521</width>
        <height>691</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CMyLabel</class>
   <extends>QLabel</extends>
   <header>cmylabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
